src/AddIns/Uno.UI.MediaPlayer.WebAssembly/MediaPlayerExtension.cs

```diff
@@ -228,7 +228,6 @@ private void InitializePlayer()
 		_owner.PlaybackSession.PlaybackStateChanged -= OnStatusChanged;
 		_owner.PlaybackSession.PlaybackStateChanged += OnStatusChanged;
 
-		ApplyAnonymousCors();
```

Comment:
Why is this removed? If someone sets the default, it may not be applied properly.